src/server/oasisapi/schemas/model_settings.json

```diff
@@ -1,217 +1,260 @@
-                    "description": "Key value pairs to present in UI",
-                    "patternProperties": {
-                        "^[a-zA-Z0-9]{3}$": {"type": "string"}
-                    },
-                    "additionalProperties": false,
-                    "minProperties": 1
-                }
-            },
-            "required": ["type", "values"]
-        }
-    },
-     "properties": {
-         "model_settings":{
-            "type": "array",
-            "uniqueItems": false,
-            "title": "Model setting options",
-            "description": "Runtime settings available to a model",
-            "items": {
-                "type": "object",
-                "properties": {
-                    "event_set":{
-                        "title": "Event set selector",
-                        "description": "The 'key' from values is used as a file suffix' events_<k>.bin",
-                        "$ref": "#/definitions/model_option_dictionary"
-                    },
-                    "event_occurrence_id":{
-                        "title": "Occurrence set selector",
-                        "description": "The 'key' from values is used as a file suffix' occurrence_<k>.bin",
-                        "$ref": "#/definitions/model_option_dictionary"
-                    }
-                },
-                "patternProperties": {
-                    "^[a-zA-Z0-9]*$": {
-                        "anyOf": [
-                            {"$ref": "#/definitions/model_option_float"},
-                            {"$ref": "#/definitions/model_option_boolean"}
-                        ]
-                    }
-                }
-            }
-        },
-        "lookup_settings":{
-            "type": "array",
-            "uniqueItems": false,
-            "title": "Model Lookup options",
-            "description": "Model lookup section",
-            "items": {
-                "type": "object",
-                "properties": {
-                    "PerilCodes":{
-                        "title": "Lookup Peril codes",
-                        "description": "Display a list of Valid OED peril codes",
-                        "$ref": "#/definitions/lookup_supported_perils"
-                    }
-                }
-            }
+    "properties": {
+        "model_settings":{
+           "type": "object",
+           "uniqueItems": false,
+           "title": "Model setting options",
+           "description": "Runtime settings available to a model",
+           "additionalProperties": false,
+           "properties": {
+               "event_set":{
+                   "title": "Event set selector",
+                   "description": "The 'key' from values is used as a file suffix' events_<k>.bin",
+                   "type": "object",
+                   "uniqueItems": false,
+                   "title": "Selection",
+                   "description": "Drop down Selection from 'options'",
```

Comment:
Why are there two descriptions

Reply:
Good spot, I needed to unpack the `definitions` sections and missed the double description